milestones.nix

```
@@ -5,9 +5,9 @@
   # Commits sourced from:
   # https://raw.githubusercontent.com/FuelLabs/fuelup/gh-pages/channel-fuel-testnet.toml
   testnet = {
-    forc-wallet = "8d59dbc462dab2d438c6ec8e52051deb000ac5a4";
-    fuel-core = "ccff696336406e842c9ed25bc23af8bcaa47eccd";
-    sway = "d821dcb0c7edb1d6e2a772f5a1ccefe38902eaec";
+    forc-wallet = "9015e09f45818bbb8068fa644e9cda0e5b5c3809";
+    fuel-core = "ba653d74fdddc397c971e5da79f189dd76ef877f";
```

```suggestion
    fuel-core = "6dc7e90305d59c04b5c6e83a6dde84abdf928ccb";
```

Should this be [6dc7e90305d59c04b5c6e83a6dde84abdf928ccb](https://github.com/FuelLabs/fuel-core/commit/6dc7e90305d59c04b5c6e83a6dde84abdf928ccb)?

---

this looks like the commit that bumped the repo to 0.43.1 to me.... what commit do you propose we put here instead?

---

Oh I see, i generally take them from the tag (https://github.com/FuelLabs/fuel-core/releases/tag/v0.43.1), that's why I was confused but seems like they backported it to master using the commit you used.

I think both of them would work no issues :)

---

makes sense! :)